WordPress/openverse
# Add pagination examples to fix DRF spectacular warnings

## Fixes



Fixes #3037 by @obulat

## Description


This PR adds examples for each field in `StandardPagination`. This removes the warning mentioned in the linked issue.

## Testing Instructions

Run the API (`just up`) and go to the documentation page (http://0.0.0.0:50280/v1/#tag/images/operation/images_search).
Look at the Docker logs: you should not see any warnings from the issue.
The paginated response examples in the documentation side bar should use the example values.
Fixed in this PR, result count is 10000, page count is 20:
<img width="554" alt="Screenshot 2023-09-19 at 10 09 51 AM" src="https://github.com/WordPress/openverse/assets/15233243/4c04e45f-a536-4378-a289-caf904ab03a5">

Current in production, result count is 1, page count is 1:
<img width="537" alt="Screenshot 2023-09-19 at 10 10 11 AM" src="https://github.com/WordPress/openverse/assets/15233243/6d58d834-7977-4d18-bedf-b8a862c783e0">


## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
